helm/cluster-operator/templates/deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   template:
     metadata:
       labels:
-        {{- include "labels.selector" . | nindent 8 }}
+        {{- include "labels.common" . | nindent 8 }}
```

Comment:
I do not really understand this but it might work. `kvm-operator` seems to miss the version in the alerts and looks the same like `cluster-operator` while `aws-operator` has the version in the alerts according to Nick and the deployment over there looks good, I guess? 

* https://github.com/giantswarm/kvm-operator/blob/master/helm/kvm-operator/templates/deployment.yaml#L21
* https://github.com/giantswarm/aws-operator/blob/master/helm/aws-operator/templates/deployment.yaml#L21